icssc-projects/peterportal-client
# Redesigned course and professor page

Implemented Design@UCI's design into PeterPortal Client.

## Description
Created a shared SideBar Component for Course and Professor Page to use.

## Steps to verify/test this change:
1. Pull changes
2. Open Course and Professor Page
3. Verify Design Changes
